src/BarChart/BarChart.xml

```diff
@@ -190,6 +190,11 @@
             <category>Appearance</category>
             <description/>
         </property>
+        <property key="refreshInterval" type="integer" defaultValue="0">
+            <caption>Refresh interval (ms)</caption>
+            <category>Appearance</category>
+            <description>Refresh the chart on milliseconds interval, when set to 0 refresh is disabled</description>
```

Comment:
This description sucks. :)

Reply:
Any ideas on a better description??.
The number of milliseconds it takes for a chart to refresh, when set to 0 refresh is disabled